src/twisted/internet/test/test_tcp.py

```diff
@@ -3010,7 +3010,7 @@ class TestException(Exception):
                 self.assertFalse(self.events)
                 raise TestException()
 
-        self.assertEqual(1, len(self.events))
+        self.assertEqual(1, len(self.events))  # type: ignore[unreachable]
```

Comment:
There are a few tests that capture exceptions using a contest manager; Mypy doesn't know that the exceptions are captured, so it things the following code is unreachable.